packages/block-editor/src/hooks/align.js

```diff
@@ -80,7 +80,7 @@ export function getValidAlignments(
 /**
  * Filters registered block settings, extending attributes to include `align`.
  *
- * @param  {Object} settings Original block settings
+ * @param {Object} settings Original block settings
  * @return {Object}          Filtered block settings
```

Comment:
```suggestion
 * @return {Object} Filtered block settings
```

Remove additional whitespace for @return descriptions